### PR TITLE
RIA-89 Cant create test from RAML using "New Functional Test" dialog

### PR DESCRIPTION
**Pull request by Alexander-Paeshin**
https://smartbear.atlassian.net/browse/RIA-89

### CODE REVIEW

File: src/main/groovy/com/smartbear/soapui/raml/RamlUtils.groovy [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        if (ramlYamlComment.equals(RAML_V08_COMMENT)) {
+            SoapUI.log(String.format(RAML_VERSION_MESSAGE, "0.8"));
+            return new RamlV08Importer(project);
+        } else if (ramlYamlComment.equals(RAML_V10_COMMENT)) {
+            SoapUI.log(String.format(RAML_VERSION_MESSAGE, "1.0"));
+            return new RamlV10Importer(project);
+        }
+
+        throw new Exception("Unable to determine the RAML version.");
```
Comment (IlyaAvdeev) [inline src/main/groovy/com/smartbear/soapui/raml/RamlUtils.groovy]: maybe it's better to use "Unknown RAML version"?

File: src/main/groovy/com/smartbear/soapui/raml/RamlV10Importer.groovy [hunk 2]
```diff
@@ -47,11 +47,15 @@ class RamlV10Importer extends AbstractRamlImporter {
     public RestService importRaml(String url) {
         RamlModelResult ramlModelResult = new RamlModelBuilder().buildApi(url)
         if (ramlModelResult.hasErrors()) {
+            StringBuilder message = new StringBuilder()
             ramlModelResult.getValidationResults().each {
-                SoapUI.log.error(it.getMessage())
+                //SoapUI.log.error(it.getMessage())
```
Comment (IlyaAvdeev) [inline src/main/groovy/com/smartbear/soapui/raml/RamlV10Importer.groovy]: why did you keep it?

File: pom.xml [hunk 3]
```diff
@@ -56,7 +56,7 @@
         <dependency>
             <groupId>com.smartbear</groupId>
             <artifactId>ready-api-soapui-pro</artifactId>
-            <version>1.2.0</version>
+            <version>1.9.0</version>
```
Comment (IlyaAvdeev) [inline pom.xml]: or even earlier, will this work for 1.5 users and later?

File: src/main/groovy/com/smartbear/soapui/raml/RamlUtils.groovy (hunk 1 above)
Comment (IlyaAvdeev) [inline src/main/groovy/com/smartbear/soapui/raml/RamlUtils.groovy]: ok, then maybe add ramlYamlComment content to the error message?

File: pom.xml (hunk 3 above)
Comment (Alexander-Paeshin) [inline pom.xml]: I've changed version to 1.8.5. I've tested how Ready! API1.5 works with the new version of the plugin and i did not found any issues.

File: src/main/groovy/com/smartbear/soapui/raml/RamlV10Importer.groovy (hunk 2 above)
Comment (Alexander-Paeshin) [inline src/main/groovy/com/smartbear/soapui/raml/RamlV10Importer.groovy]: Fixed

**Pull request merged**